client/app/components/routes/select-operator/form/types.ts

```diff
@@ -0,0 +1,22 @@
+export interface Operator {
+  id: number;
+  legal_name: string;
+  trade_name: string;
+  cra_business_number: string;
+  bc_corporate_registry_number: string;
+  duns_number: number;
+  business_structure: string;
+  physical_street_address: string;
+  physical_municipality: string;
+  physical_province: string;
+  physical_postal_code: string;
+  mailing_street_address: string;
+  mailing_municipality: string;
+  mailing_province: string;
+  mailing_postal_code: string;
+  bceid: number;
+  website: string;
+  compliance_entity: number;
```

Comment:
I don't think we need the `compliance_entity` or `duns_number` fields anymore...? I'm almost certain we don't need compliance_entity, anyway